bloch_sphere/animate_bloch.py

```diff
@@ -142,6 +143,15 @@ def apply_gate_list(self, gates, final_wait=True):
                     print(f'Error: Custom gate arguments contain invalid '
                           f'floats {gate}.')
                 continue
+            if re.match('R[x,y,z][,;]', gate[:3]):
+                try:
+                    x, y, z = gate[1] == 'x', gate[1] == 'y', gate[1] == 'z' 
+                    r_pi = float(gate[3:])
+                    label = f"{gate[:2]}({r_pi:.3f}".rstrip('0').rstrip('.') + ")"
+                    self.custom_gate(x, y, z, r_pi, label=label)
+                except ValueError:
```

Comment:
Put the try/catch around the fewest possible lines to catch the float parsing error.  Either the r_pi line or the custom_gate line.

Reply:
Done